HLTrigger/Muon/plugins/HLTMuonL1TFilter.h

```diff
@@ -38,6 +38,10 @@ class HLTMuonL1TFilter : public HLTFilter {
   /// pT threshold
   double minPt_;
 
+  /// max dRs for L1 candidate matching
+  double maxDR_;
```

Comment:
If you decide to touch the code, these and all other private class members can also be made `const`